CMakeLists.txt

```diff
@@ -222,6 +222,14 @@ Please see https://github.com/love2d/megasource
 		${ZLIB_LIBRARY}
 	)
 
+	if(MINGW)
+		set(LOVE_LINK_LIBRARIES
+			mingw32
+			SDL2main

```

Comment:
I was curious why it wasn't already linking with SDL2main so I did some googling (more than I would have liked to do), and I think it might work to change the include dirs and link libraries above this from `SDL2_INCLUDE_DIR` and `SDL2_LIBRARY` to `SDL2_INCLUDE_DIRS` and `SDL2_LIBRARIES`, in order to link with SDL2main on windows?

If that works I think it'd be good to change this to do that instead, since it's cleaner.

Reply:
That does work, however one needs to configure `find_package` to look for the `SDL2main`. I oriented myself by [SDL2's README for cmake](https://wiki.libsdl.org/SDL2/README/cmake#including-sdl-in-your-project). Since the config has no `REQUIRED` attribute, it should not break anything on other plattforms, I suppose.
Adjusted in 11d4bdd.